src/System.Private.Uri/tests/FunctionalTests/IdnDnsSafeHostTest.cs

```diff
@@ -45,6 +45,21 @@ public void IdnDnsSafeHost_IPv6Host_ScopeIdButNoBrackets()
             Assert.Equal("[::1]", test.Host);
         }
 
+        [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        [InlineData("fe80::e077:c9a3:eeba:b8e9", "%18")]
+        public void IdnDnsSafeHost_IPv6HostLinkLocalAddress_ScopeIdButNoBrackets(string address, string zoneIndex)
```

Comment:
"ScopeIdButNoBrackets" part of the test name isn't quite right since there is an Assert in this test that validates for brackets.

Normally we don't have a single test method that test multiple properties.  But I see that there is already other tests that do this in this file.  

I suggest, though, that you rename the test like this:

"IdnDnsSafeHost_IPv6HostLinkLocalAddress_ScopeIdCorrectlyFormatted"